src/client/pmix_client_get.c

```diff
@@ -615,7 +615,18 @@ static void _getnb_cbfunc(struct pmix_peer_t *pr,
                                 "pmix: get_nb searching for key %s for proc %s",
                                 cb->key, PMIX_NAME_PRINT(&proc));
             PMIX_GDS_FETCH_KV(rc, pmix_globals.mypeer, cb);
-            if (PMIX_SUCCESS == rc) {
+            if (PMIX_SUCCESS != rc) {
```

Comment:
Just curious - in the master branch, we dealt with the case where `PMIX_OPERATION_SUCCEEDED` was returned, but here we don't. Is that an oversight, or does this release branch never return that status?

Reply:
That I don't know, I was wondering the same thing, but have not looked into it.

Reply:
Probably safer to add the check - worst case it does nothing (as the hash never returns that code). However, if we ever backport something to the hash module that _does_ return "succeeded", it will save us chasing our tails trying to figure out what broke.